### PR TITLE
updated TOC, cleaned up configurator stuff

### PR DESCRIPTION
**Pull request by benjaminleesmith**

### CODE REVIEW

File: tap-gui/configurator/create-plug-in-wrapper.hbs.md [hunk 1]
```diff
@@ -491,4 +491,5 @@ $ yarn tsc && yarn build
 ## Next steps
 
 You should be ready to publish your Tanzu Developer Portal plug-ins to the registry of your choosing.
-Once you have published them, you can [build a customized Tanzu Developer Portal with Configurator](building.hbs.md).
+If you are publishing your plug-ins to a private registry, see the[Configure the Configurator with a private registry](private-registries.hbs.md) tutorial.
```
Comment (weresch) [inline tap-gui/configurator/create-plug-in-wrapper.hbs.md:494]: Needs a space between the[Configure

Comment (mstergianis) [inline tap-gui/configurator/create-plug-in-wrapper.hbs.md:494]:
```suggestion
If you are publishing your plug-ins to a private registry, see the [Configure the Configurator with a private registry](private-registries.hbs.md) tutorial.
```

Comment (benjaminleesmith) [reply tap-gui/configurator/create-plug-in-wrapper.hbs.md]: fixed thanks

**Comment by benjaminleesmith** [comment]
> Hi @benjaminleesmith, does this only apply to 1.8 or does it apply to 1.7 as well?

This applied to 1.7 and 1.8. Thanks!

**Pull request merged**